device-types/Cisco/DS-C9148T-48PITK9.yml

```diff
@@ -0,0 +1,136 @@
+---
+manufacturer: Cisco
+model: MDS 9148T 48X32G FC
+slug: cisco-ds-c9148t-48pitk9
+part_number: DS-C9148T-48PITK9
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight: 16.7
+weight_unit: lb
+comments: '[Cisco MDS 9148T 32-Gbps 48-Port Fibre Channel Switch Data Sheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148t-32-gbps-48-port-fibre-channel-switch/data-sheet-c78-740623.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+module-bays:
+  - name: Slot 1
+    position: '1'
+    description: '4/8/16/32 Gbps FC/Sup-4'
+  - name: Slot 2
+    position: '2'
+    description: 'MDS 9148T 48X32G FC (1 RU) Chassis Power Supply'
+  - name: Slot 3
+    position: '3'
+    description: 'MDS 9148T 48X32G FC (1 RU) Chassis Power Supply'
+  - name: Slot 4
+    position: '4'
+    description: 'MDS 9148T 48X32G FC (1 RU) Chassis Fan Module'
+  - name: Slot 5
+    position: '5'
+    description: 'MDS 9148T 48X32G FC (1 RU) Chassis Fan Module'
+  - name: Slot 6
+    position: '6'
+    description: 'MDS 9148T 48X32G FC (1 RU) Chassis Fan Module'
+  - name: Slot 7
+    position: '7'
+    description: 'MDS 9148T 48X32G FC (1 RU) Chassis Fan Module'
```

Comment:
Is this what shows in `show inventory`?

I know on our Nexus 93180YC-FX3 it is not...

```
# show inventory
NAME: "Chassis",  DESCR: "Nexus9000 C93180YC-FX3 Chassis"
PID: N9K-C93180YC-FX3    ,  VID: V01 ,  SN: ###########

NAME: "Slot 1",  DESCR: "48x10/25G + 6x40/100G Ethernet Module"
PID: N9K-C93180YC-FX3    ,  VID: V01 ,  SN: ###########

NAME: "Power Supply 1",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Power Supply"
PID: NXA-PAC-650W-PI     ,  VID: V04 ,  SN: ###########

NAME: "Power Supply 2",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Power Supply"
PID: NXA-PAC-650W-PI     ,  VID: V04 ,  SN: ###########

NAME: "Fan 1",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A

NAME: "Fan 2",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A

NAME: "Fan 3",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A

NAME: "Fan 4",  DESCR: "Nexus9000 C93180YC-FX3 Chassis Fan Module"
PID: NXA-FAN-35CFM-PI    ,  VID: V01 ,  SN: N/A
```

Reply:
# show inventory
NAME: "Chassis",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis "   
PID: DS-C9148T-K9        ,  VID: V02 ,  SN: XXXXXXXXXXX          

NAME: "Slot 1",  DESCR: "4/8/16/32 Gbps FC/Sup-4"               
PID: DS-C9148T-K9-SUP    ,  VID: V02 ,  SN: XXXXXXXXXXX

NAME: "Slot 2",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis Power Supply"
PID: DS-CAC-650W-E       ,  VID: V00 ,  SN: XXXXXXXXXXX

NAME: "Slot 3",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis Power Supply"
PID: DS-CAC-650W-E       ,  VID: V00 ,  SN: XXXXXXXXXXX

NAME: "Slot 4",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis Fan Module"
PID: DS-C32T-FAN-E       ,  VID: V00 ,  SN: N/A                  

NAME: "Slot 5",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis Fan Module"
PID: DS-C32T-FAN-E       ,  VID: V00 ,  SN: N/A                  

NAME: "Slot 6",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis Fan Module"
PID: DS-C32T-FAN-E       ,  VID: V00 ,  SN: N/A                  

NAME: "Slot 7",  DESCR: "MDS 9148T 48X32G FC (1 RU) Chassis Fan Module"
PID: DS-C32T-FAN-E       ,  VID: V00 ,  SN: N/A